pom.xml

```diff
@@ -630,7 +630,7 @@
                 <plugin>
                     <groupId>org.codehaus.mojo</groupId>
                     <artifactId>exec-maven-plugin</artifactId>
-                    <version>1.5.0</version>
+                    <version>1.6.0</version>
```

Comment:
❓ Is this upgrading the `exec-maven-plugin` for every subproject? What was the issue warranting this as part of this large PR?

Reply:
Well for one there is a new version. And for two, I ran into issues with it and solr for some reason and bumping it seemed to make it go away.